guides/10_Video_Processor.md

```diff
@@ -145,6 +145,41 @@ async process(buffers: VideoFrameBuffer[]): Promise<VideoFrameBuffer[]> {
 }
 ```
 
+### Building an overlay processor
+An overlay processor can be a customized processor for loading an external image:
+
+```typescript
+class VideoLoadImageProcessor implements VideoFrameProcessor { 
+  private targetCanvas: HTMLCanvasElement = document.createElement('canvas') as HTMLCanvasElement;
+  private targetCanvasCtx: CanvasRenderingContext2D = this.targetCanvas.getContext('2d') as CanvasRenderingContext2D;
+  private canvasVideoFrameBuffer = new CanvasVideoFrameBuffer(this.targetCanvas);
+
+  private renderWidth: number = 0;
+  private renderHeight: number = 0;
+  private sourceWidth: number = 0;
+  private sourceHeight: number = 0;
+  
+  async function loadImage(url: string, elem: HTMLImageElement): Promise<HTMLImageElement> {
```

Comment:
We have the function here, but should we explain or add a bit on where to add this function as an listener or comment for clarifying?